package.json

```diff
@@ -21,7 +21,8 @@
       "node_modules/**/*"
     ],
     "asarUnpack": [
-      "build/bin/legendary",
+      "build/bin/linux/legendary",
+      "build/bin/darwin/legendary",
```

Comment:
Since the same binary works for both, why do we need to duplicate it?

Reply:
It's not the same binary actually, linux binary does not even launch on mac. I built mac one from legendary source using pyinstaller

Reply:
Ah, I didn't know that, I guess I missed this part on your explanation. 
Nice then. :D